api/eds.proto

```diff
@@ -83,6 +83,16 @@ message LbEndpoint {
   // LocalityLbEndpoints. If unspecified, each host is presumed to have equal
   // weight in a locality.
   google.protobuf.UInt32Value load_balancing_weight = 4;
+
+  // Optional: the priority for this LbEndpoint
```

Comment:
Maybe point out that if not specified, this is the proto3 default value of 0 and the endpoints will be highest priority.